packages/webdriverio/src/commands/element/click.ts

```diff
@@ -8,8 +8,13 @@ const log = logger('webdriverio/click')
  *
  * Click on an element.
  *
- * Note: This issues a WebDriver `click` command for the selected element, which generally scrolls to and then clicks the
- * selected element. However, if you have fixed-position elements (such as a fixed header or footer) that cover up the
+ * This issues a WebDriver `click` command for the selected element , which generally scrolls to and then clicks the
+ * selected element when no options are passed. When options object is passed it uses action class instead of webdriver click which
+ * give added capabilities like passing button type, coordinates etc. By default, when using options a release action
+ * command is send after performing the click action, this could cause some alerts to be closed unexpectedly
+ * (issue: https://github.com/webdriverio/webdriverio/issues/6423) To fix this pass `option.skipRelease`.
```

Comment:
I am not sure if it is worth mentioning this here because we don't really know what was responsible for the alert to disappear and if this case is true for all similar scenarios.

Reply:
I think we can simply say, if this flag is set WebdriverIO won't call the [`releaseAction`](https://w3c.github.io/webdriver/webdriver-spec.html#releasing-actions) command